src/backend/sgx/builder.rs

```diff
@@ -50,12 +51,11 @@ impl TryFrom<super::config::Config> for Builder {
             map.addr() + map.size()
         );
 
-        // Open the device.
-        let mut file = OpenOptions::new()
-            .read(true)
-            .write(true)
-            .open("/dev/sgx_enclave")
-            .context("Failed to open '/dev/sgx_enclave'")?;
+        // Create a new enclave instance
+        let mut file = Enclave::new()
+            .context(format!("Failed to open '{}'", env!("SGX_ENCLAVE_DEVICE")))
```

Comment:
We probably want a constant here (https://github.com/enarx/enarx/pull/1599/files#r837508756)